hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -551,7 +551,88 @@
       default:   "5",
     },
   ],
+  features: [
+    {
+     name: "FLASH_CTRL.ESCALATION"
+     desc:'''Flash controller has two sources of escalation.
+     1. Global escalation : This is initiated by the LC (Lifecycle Controller) through the use of lc_escalate_en.
+     2. Local escalation : This is activated in response to standard faults detected within the flash.
+                           These faults are monitored through flash_ctrl.STD_FAULT_STATUS.
+     '''
+    }
+    {
+      name: "FLASH_CTRL.FETCH_CODE"
```

Comment:
@moidx @timothytrippel I see all test_rom tests are enabled fetch_code mode in
[here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/test_rom/test_rom.c;drc=2741ba12c2f61b9fabba61f5bf21f540d94947c2;l=201).
So I assume this is covered in test_rom mode. Do we need to create dedicated test for Sival?

Reply:
For SiVal, tests will run with the production ROM, since that is what the chip will actually have, but the ROM also enables flash execution once the signature(s) have been checked: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/lib/sigverify/rsa_verify.c;drc=7d15a8d910a7de5aa7751a263cbbe55a6f6f5990;l=72

So we should be good.